### PR TITLE
Feat: Added test flow drag drop functionality

### PR DESCRIPTION
**Pull request by anujnema5**
## Description

#1398 
In this pr i have added the functionality of drag and drop in the test flow 

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/packages/@workspaces/features/collection-list/components/request/Request.svelte [hunk 1]
```diff
@@ -121,6 +121,19 @@
...
+
+  function dragStart(event, collection) {
+    const data = {
+      workspaceId: collection.workspaceId,
+      collectionId: collection.id,
+      folderId: folder?.id ?? "",
+      requestId: api.id,
+      name: api.name,
+      method: api.request.method,
+      collection: collection,
```
Comment (Astitva877) [inline src/packages/@workspaces/features/collection-list/components/request/Request.svelte:133]: from here we don't need to send the collection, it's already available in testflowexplorer

File: src/packages/@workspaces/features/testflow-explorer/layout/TestflowExplorer.svelte [hunk 2]
```diff
@@ -298,6 +323,58 @@
     if (val) onUpdateEdges(val);
   });
 
+  function drop(event) {
```
Comment (Astitva877) [inline src/packages/@workspaces/features/testflow-explorer/layout/TestflowExplorer.svelte:326]: convert this to arrow function

**Review by Astitva877 — CHANGES_REQUESTED** [review]
Please resolve the requested changes.

File: src/packages/@workspaces/features/testflow-explorer/layout/TestflowExplorer.svelte [hunk 3]
```diff
@@ -40,6 +40,17 @@
...
   export let toggleHistoryContainer;
 
+  let folderId: string;
+  let workspaceId: string;
+  let collectionId: string;
+  let id: string;
+  let method: string;
+  let name: string;
+  let collection;
+  let requestId: string;
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/testflow-explorer/layout/TestflowExplorer.svelte:50]: data type missing

File: src/packages/@workspaces/features/testflow-explorer/layout/TestflowExplorer.svelte [hunk 4]
```diff
@@ -298,6 +323,58 @@
...
+
+    if (!data) return;
+
+    id = data.apiId;
+    folderId = data.folderId;
+    workspaceId = data.workspaceId;
+    collectionId = data.collectionId;
+    name = data.name;
+    method = data.method;
+    collection = data.collection;
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/testflow-explorer/layout/TestflowExplorer.svelte:338]: add let

**Pull request closed without merging**